app/components/models/contivglobals.ts

```diff
@@ -44,6 +44,7 @@ export const ContivGlobals = {
     'NUMBER_REGEX' : '^[0-9]*$',
     'USERNAME_REGEX': /^([a-zA-Z0-9\_\-\.\@])+$/,
     'LDAPGROUP_REGEX': /([\=])+/g,
+    'NETWORK_NAME_REGEX': /^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\-]*[A-Za-z0-9])$/,
```

Comment:
Where did you get the requirements for this regex?

Also, you can remove the A-Z entries and just put a `i` on the end to make the regex case insensitive, e.g.,:

```js
/^(([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])\.)*([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])$/i
```

Saves a lot of typing lol